src/components/RightSidebar/Description/Description.vue

```diff
@@ -0,0 +1,423 @@
+		},
+
+		showOverlay() {
+			return this.overflows && !this.editing && !this.loading && !this.expanded
+		},
+	},
+
+	watch: {
+		// Each time the prop changes, reflect the changes in the value stored in this component
+		description() {
+			this.descriptionText = this.description
+			if (!this.editing) {
+				this.checkOverflow()
+			}
+		},
+		editing(newValue) {
+			if (!newValue) {
+				this.descriptionText = this.description
+			}
+		},
+	},
+
+	methods: {
+		handleEditDescription() {
+			const contenteditable = this.$refs.contenteditable.$refs.contenteditable
+			this.$emit('update:editing', true)
+			this.$nextTick(() => {
+				// Focus and select the text in the description
+				contenteditable.focus()
+				document.execCommand('selectAll', false, null)
+			})
+		},
+
+		handleSubmitDescription() {
+			if (!this.canSubmit) {
+				return
+			}
+			// Remove newlines and whitespaces.
+			this.descriptionText = this.descriptionText.replace(/\r\n|\n|\r/gm, '').trim()
+			// Submit description
+			this.$emit('submit:description', this.descriptionText)
+			/**
+			 * Change the richcontenteditable key in order to trigger a re-render
+			 * without this all the trimmed new lines and whitespaces would
+			 * still be present in the contenteditable element.
+			 */
+			this.forceReRenderKey += 1
+		},
+
+		handleCancelEditing() {
+			this.descriptionText = this.description
+			this.$emit('update:editing', false)
+			// Deselect all the text that's been selected in `handleEditDescription`
+			window.getSelection().removeAllRanges()
+		},
+
+		// Expand the description
+		handleClick() {
+			if (this.editing || this.loading) {
+				return
+			} if (this.overflows) {
+				this.expanded = !this.expanded
+			}
+		},
+
+		// Collapse the description or dismiss editing
+		handleClickOutside() {
+			this.expanded = false
+			this.$emit('update:editing', false)
+		},
+
+		checkOverflow() {
+			const descriptionScrollHeight = this.$refs.description.offsetHeight
+			const descriptionOffsetHeight = this.$refs.contenteditable.$refs.contenteditable.offsetHeight
+			this.overflows = descriptionScrollHeight > descriptionOffsetHeight
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../../../assets/variables.scss';
+
+.description {
+	margin: -20px 0 8px 8px;
+	display: flex;
+	width: 100%;
+	overflow: hidden;
+	position: relative;
+	max-height: calc(var(--default-line-height) * 3 + 28px);
+	&--editing {
+		box-shadow: 0 2px var(--color-primary-element);
+		transition: all 150ms ease-in-out;
+		max-height: unset;
+		align-items: flex-end;
+	}
+	&--expanded {
+		max-height: unset;
+		min-height: $clickable-area * 2;
+		align-items: flex-end;
+	}
+	&__header {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+		height: 44px;
+	}
+	&__title {
+		color: var(--color-primary);
+		font-weight: bold;
+		font-size: var(--default-font-size);
+		line-height: var(----default-line-height);
```

Comment:
```suggestion
		line-height: var(--default-line-height);
```

@marcoambrosini :see_no_evil:

Reply:
Fix at https://github.com/nextcloud/spreed/pull/7185

Reply:
oops